test/parallel/test-exec-max-buffer.js

```diff
@@ -9,3 +9,14 @@ exec(cmd, { maxBuffer: 5 }, function(err, stdout, stderr) {
   assert.ok(err);
   assert.ok(/maxBuffer/.test(err.message));
 });
+
+if (!common.isWindows) {
```

Comment:
Did you try @bnoordhuis suggestion? If that doesn't work, you should add a comment on why this is skipped on Windows.